### PR TITLE
Add DSA fields and validation

### PR DESCRIPTION
**Pull request by AntoxaAntoxic**
https://github.com/prebid/prebid-server/issues/3367

### CODE REVIEW

**Review by SerhiiNahornyi — DISMISSED** [review]

**Review by SerhiiNahornyi — APPROVED** [review]

**Review by Net-burst — DISMISSED** [review]
LGTM

File: src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRegsDsa.java [hunk 1]
```diff
@@ -11,9 +11,9 @@
 public class ExtRegsDsa {
 
     /**
-     * Defines the contract for bidrequest.regs.ext.dsa.required
+     * Defines the contract for bidrequest.regs.ext.dsa.dsarequired
      */
-    Integer required;
+    Integer dsarequired;
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRegsDsa.java:16]: Please follow Java code style and use @JsonProperty or use Jackson naming strategy

File: src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRegsDsaTransparency.java [hunk 2]
```diff
@@ -16,7 +16,7 @@ public class ExtRegsDsaTransparency {
     String domain;
 
     /**
-     * Defines the contract for bidrequest.regs.ext.dsa.transparency[i].params[]
+     * Defines the contract for bidrequest.regs.ext.dsa.transparency[i].dsaparams[]
      */
-    List<Integer> params;
+    List<Integer> dsaparams;
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRegsDsaTransparency.java:21]: Please follow Java code style and use @JsonProperty or use Jackson naming strategy

File: src/test/groovy/org/prebid/server/functional/model/request/auction/DsaTransparency.groovy [hunk 3]
```diff
@@ -7,7 +7,7 @@ import org.prebid.server.functional.util.PBSUtils
 class DsaTransparency {
 
     String domain
-    List<Integer> params
+    List<Integer> dsaparams
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/model/request/auction/DsaTransparency.groovy:10]: Please follow Java code style and use @JsonProperty or use Jackson naming strategy

File: src/test/groovy/org/prebid/server/functional/model/request/auction/RegsDsa.groovy [hunk 4]
```diff
@@ -9,14 +9,14 @@ import org.prebid.server.functional.util.PBSUtils
 @ToString(includeNames = true, ignoreNulls = true)
 class RegsDsa {
 
-    Integer required
+    Integer dsarequired
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/model/request/auction/RegsDsa.groovy:12]: Please follow Java code style.

**Review by Net-burst — CHANGES_REQUESTED** [review]

**Review by Net-burst — APPROVED** [review]
LGTM

**Pull request merged**